### ISSUE TITLE
Integrate Protobuf serde

### ISSUE THREAD

**Issue by carlesarnal** [quarkusio/quarkus#37880]
Description
Integrate Protobuf serializer and deserializer for Kafka.
Initially, this work should use the Apicurio Protobuf serde but also use the Confluent alternative.

**Comment by carlesarnal** [comment]
https://github.com/quarkusio/quarkus/pull/37981

**Comment by cescoffier** [comment]
@carlesarnal @alesj any update on this?

**Comment by carlesarnal** [comment]
> [@carlesarnal](https://github.com/carlesarnal) [@alesj](https://github.com/alesj) any update on this?

No, sorry. I've been dragged to other priorities and I did not had time to look at the existing issue. The integration is functional in JVM mode but there are some substitutions likely required for the native mode to work. I will re-visit this as soon as I can.

**Comment by cescoffier** [comment]
@carlesarnal Any news on this?

**Comment by carlesarnal** [comment]
We need to move the extensions to the Quarkiverse, I have created https://github.com/quarkusio/quarkus/issues/54373 and https://github.com/quarkusio/quarkus/issues/54374.

**Comment by gastaldi** [comment]
Extensions moved to https://github.com/quarkiverse/quarkus-registry-protobuf
